filesystem/lvm2.ksy

```diff
@@ -0,0 +1,164 @@
+meta:
+  title: Logical Volume Manager version 2
+  id: lvm2
+  endian: le
+  encoding: ascii
+  application:
+    - linux
+    - grub2
+    - lvm tools
+    - libvslvm
+  license: GFDL-1.3+
+doc: |
+  === Building a test file
+  ....
+  dd if=/dev/zero of=image.img bs=512 count=$(( 4 * 1024 * 2 ))
+  sudo losetup /dev/loop1 image.img
+  sudo pvcreate /dev/loop1
+  sudo vgcreate vg_test /dev/loop1
+  sudo lvcreate --name lv_test1 vg_test
```

Comment:
BTW, this command returns on several systems where I've tried that:

```
  No command with matching syntax recognised.  Run 'lvcreate --help' for more information.
```

Probably the correct one is something like:

```
sudo lvcreate --size=4M --name lv_test1 vg_test
```

Also, at least on the systems I've tested, minimal extent size is 4MiB, and it's impossible to create LV on a PV that totals 4MiB (as you use `dd` which creates 4MiB image.img). Using a larger file fixes that problem as well.